daml.yaml

```diff
@@ -1,7 +1,9 @@
 sdk-version: 2.4.0
 name: daml-ctl
-version: 2.2.2
+version: 2.2.3
```

Comment:
@georg-da what version should we use?

- it is a breaking change for previous clients, so probably `3.0.0`
- on the other hand, I'm not sure `1.15` is stable, so probably we should not be releasing an official version

Most likely these are just philosophical questions, as I guess no one is really using `daml-ctl` alone in production

Reply:
Yeah, fair point. Since all of this is considered pre-(stable-)release, I'm ok with just bumping minor.